TestCases/compliance-level-3/0066-feel-negation/0066-feel-negation-test-01.xml

```diff
@@ -32,23 +32,23 @@
     </testCase>
 
     <testCase id="003_b">
-        <resultNode name="decision003_b" type="decision">
+        <resultNode errorResult="true" name="decision003_b" type="decision">
```

Comment:
Case:
https://github.com/dmn-tck/tck/blob/9f8ba50870259c971b5980d031fe73224635be09/TestCases/compliance-level-3/0066-feel-negation/0066-feel-negation.dmn#L40

DMN Spec:

> Parameter Domain: boolean 

Concur in the result, but I believe is missing the errorFlag to clarify this is an error case